website/docs/podman/podman-remote.md

```diff
@@ -87,6 +105,12 @@ $ podman system connection add my-remote-machine --identity ~/.ssh/id_ed25519 ss
 $ podman system connection add my-remote-machine --identity ~/.ssh/id_ed25519 ssh://root@my-server-ip/run/podman/podman.sock
 ```
 
+:::warning
+
+On Windows, you need to use **an absolute path** for the identities, path with `~` will not resolve
```

Comment:
On Windows, you need to use an absolute path for the identities; a path with `~` will not resolve.

Reply:
We should reserve bold formatting only for UI elements, not to highlight something in the sentence. The info is highlighted as it is already present in a warning admonition.